test/integration/concrete/lockup/status-of/statusOf.tree

```diff
@@ -2,12 +2,12 @@ statusOf.t.sol
 ├── when the id references a null stream
 │  └── it should revert
 └── when the id does not reference a null stream
-   ├── when assets have been fully withdrawn
+   ├── given assets have been fully withdrawn
    │  └── it should return DEPLETED
-   └── when assets have not been fully withdrawn
-      ├── when the stream has been canceled
+   └── given assets have not been fully withdrawn
+      ├── given the stream has been canceled
       │  └── it should return CANCELED
-      └── when the stream has not been canceled
+      └── given the stream has not been canceled
          ├── when the start time is in the future
```

Comment:
Should be "given when the start time is in the future"

And same for the "given when the refundable amount is zero" branch